Vision/Vision.bs

```diff
@@ -159,11 +159,10 @@ Markup Shorthands: markdown yes
 	* **Open Participation**: We welcome individuals and organizations of all sizes
 		(from single-person companies to multi-nationals),
 		and take feedback from the general public.
-	* **Interoperability**: We believe proven interoperable implementation is a 
-		requirement for broad adoption of standards. 
-		To ensure reliable interoperability, 
-		we require multiple implementations 
-		and open test suites for our standards.
+	* **Interoperability**: We strive to prove the fitness of our specifications
```

Comment:
"We strive to prove the fitness of our specifications" is a convoluted phrase, and I particularly wonder whether that makes it harder for an ESL reader to understand.

Reply:
verify sgtm